.github/workflows/job_tests.yml

```diff
@@ -0,0 +1,35 @@
+name: job_tests
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  test-hashes:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: "rwth-i6/i6_core"
+          path: "i6_core"
+      - uses: actions/checkout@v2
+        with:
+          repository: "rwth-i6/sisyphus"
+          path: "sisyphus"
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
```

Comment:
Shouldn't we test also with python 3.6.9?

Reply:
As far as I remember the recipe code we are using includes some 3.7 features, so at some point 3.6.9 might not work.